client/src/SessionController.js

```diff
@@ -116,70 +117,71 @@ export function SessionController({ children }) {
 
   useEffect(() => {
     if (userStates.status === REQUEST_STATE.LOGGINGIN) {
+      dispatch(fetchLimitLeftAsync())
```

Comment:
is this supposed to be assigned to smt? seems like `fetchLimitLeftAsync()` is supposed to return smt, but it's not assigned to any variable here or w/e

Reply:
it's just being called to fetch the rate limit from the header of the user's request into the userslice, dont need to do it in the actual component